docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+
+    .. code-block:: text
+
+       kind create cluster --name trino
+
+    .. note::
+
+        The ``name`` parameter here is optional but is just used to showcase
+        how the namespace is applied in future commands. The cluster name
+        defaults to ``kind`` if no parameter is added. We recommend using
+        ``trino`` to make the application on this cluster obvious.
+
+* Install the `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`_
+  binary. Ensure you have a kubectl version that adheres to the
+  `Kubernetes version skew policy <https://kubernetes.io/releases/version-skew-policy/>`_.
+* Install the `helm <https://helm.sh>`_ binary. Ensure that you have a helm
+  version that adheres to the `Helm version skew policy <https://helm.sh/docs/topics/version_skew/>`_ that
+  supports the version of Kubernetes you are running.
+
+Running Trino using Helm
+------------------------
+
+Run the following commands from the system with ``helm`` and ``kubectl``
+installed and configured to connect to your running Kubernetes cluster:
+
+#. Validate kubectl is pointing to the correct cluster by running the command:
+
+   .. code-block:: text
+
+      kubectl cluster-info --context=kind-trino
+
+   .. note::
+
+      The ``context`` parameter here is optional and references the cluster
+      created in the example using kind above. ``kubectl`` will use the
+      `default context <https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration>`_
+      if this parameter isn't supplied. Notice the context is the name of the
+      cluster with the ``kind-`` prefix added.
+
+   You should see similar output with a random port assigned if you are
+   running on kind.
+
+   .. code-block:: text
+
+      Kubernetes control plane is running at https://127.0.0.1:59553
+      CoreDNS is running at https://127.0.0.1:59553/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+#. (Optional) Create ``values.yaml`` file to define the parameters of your
+   deployment.
+
+   The helm chart has a `set of default values <https://github.com/trinodb/charts/blob/main/charts/trino/values.yaml>`_
+   that are used if you don't specify your own ``values.yaml`` file. When you
+   create this file, you will override only the values you specify. The
+   remaining properties will remain as the default. Add a ``values.yaml``
+   with the following configuration:
+
+   .. code-block:: yaml
+
+      image:
+        tag: "|trino_version|"
+      server:
+        workers: 2
+        config:
+          memory:
+            heapHeadroomPerNode: "1GB"
+          query:
+            maxMemory: "4GB"
+            maxMemoryPerNode: "1GB"
+      coordinator:
+        jvm:
+          maxHeapSize: "8G"
+      worker:
+        jvm:
+          maxHeapSize: "8G"
+
+   Let's explain what these values do and some useful notes.
+
+   #. ``.Values.image.tag`` is set to the current version, |trino_version|. Set
```

Comment:
Is the `.Values` key valid in this context?

Reply:
hmm not sure...this is how I see it used when describing Values configs in other Kubernetes docs so I was copycatting but I am happy to remove as most will likely get confused by this. Thoughts @mosabua ?

Reply:
This is how they're referenced in the chart itself, but here you refer to keys in the `values.yaml` file directly.